system-int-test/test/cmr/system_int_test/bootstrap/bulk_index_test.clj

```diff
@@ -690,3 +694,85 @@
       (testing "All tag parameters with XML references"
         (is (d/refs-match? [coll1]
                            (search/find-refs :collection {:tag-key "tag1"})))))))
+
+(deftest bulk-index-variables-for-provider
```

Comment:
Done.

Since the file was large, and variables were only a small part of it, I split it up multiple ways.